### PR TITLE
Page type home

### PR DESCRIPTION
**Pull request by RobTobias123**
Principles moved above example of Home page-type example. The initial version of Content Guidance created and added below that, referencing the panels/rows on the examples page layout, with links to components' documentation.

### CODE REVIEW

**Review by andij — COMMENTED** [review]
Looks like the package.json has been updated. I wouldn't expect this to be part of this PR.

File: docs/page-types/home.md [hunk 1]
```diff
@@ -3,34 +3,66 @@ title: Home
...
+| :--- | :--- | :--- |
+| Warm and friendly  | Be active, Be natural | Promotional, Navigational, Informative, Memorable |
 
 ## Example
 
-https://www.britishgas.co.uk/
+<StorybookPage story="examples-page-types--home"></StorybookPage>
 
-## Principles to follow
+## Content guidance
```
Comment (PtitBen) [inline docs/page-types/home.md:31]: Should we have a mention of the ns-skyline for important messaging of "point-in-time" events ?

File: docs/page-types/home.md [hunk 2]
```diff
@@ -3,34 +3,66 @@ title: Home
...
-- Promote the latest offers.
-- Act as a ‘meeting point’, which a user can return to at any point during their interaction with the site.
+* An introduction to the British Gas brand, leaving a lasting first impression.
+* Promote specific products or services offered by British Gas.
+* Drive traffic to strategically important products and services.
+* Provide a starting point for customers to easily find what they need.
+* Promote the latest offers.
+* Act as a ‘meeting point’, which a user can return to at any point during their interaction with the site.
+
+## Principles to follow
```
[the remark this answers is not in the document]
Comment (andij) [reply docs/page-types/home.md:21]: I like this comment. My response was going to be - Shall we link to 'Warm and friendly' in the 'Creative principles'?

So I took a look at the Creative principles to see if this information is documented there, but it's not 😕  https://nucleus.design/docs/getting-started/principles#creative-principles

I tried to find a description on Frontify too, but no joy: https://centrica.frontify.com/d/6307mViOlfHB/visual-identity#/our-visual-identity/start-discovering/our-creative-principles

As there's no information of how to make something Warm and friendly, we can't do that either.

**Comment by RobTobias123** [comment]
Following discussion with Drew, I've updated some of the text for content guidance (still a bit of finessing to do) and created/added thumbnails for the visual association. Drew - I'll catch up with you tomorrow AM as discussed. Thanks for the input.

**Comment by andij** [comment]
I've introduced a table to contain the thumbnail images and content guidance.

File: docs/page-types/home.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (PtitBen) [reply docs/page-types/home.md:21]: I didn't see your comment Drew 😱

Yes, and yes! 
I think those were conversations and things that were said when workshopping and reviewing work with the old brand team, and Rufus Leonard.

Saw your ticket raised, perfect!

**Review by andij — APPROVED** [review]

**Pull request merged**